scripts/package-cli

```diff
@@ -19,6 +19,7 @@ cni_binaries=(
     "flannel"
     "host-local"
     "loopback"
+    "macvlan"
```

Comment:
@brandond since you said earlier that you decided to not add `macvlan` right now, is there a reason to import here?